wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+- If _Quit_: [Auto][Auto wikilink] will take over the _Quit_ player.
+
+#### Player box colour
+
+<!-- A player box reference here? Old images in ./img/Playerbox -->
+<!-- Special player box note: Skipped (has its own special callout), Failed (red text), and Quit (red text with [Quit] appended) -->
+
+Status/Colour | Description
+:---:|:---
+**Normal/Blue**       | **Player has _more_ than half of health**, usually at full. The concentration of blue colour will grow brighter if the player's health bar is increasing to full and bleed to red by losing the health bar more than half of the health bar length.
+**Danger/Purple-Red** | **Player has _less_ than half of health**. The concentration of red colour will grow brighter if the player's health bar is decreasing to empty and diminished to blue by filling the health bar more than half the health bar length.
+**Failed/Gray**       | **Player unable to reserve the final sliver of health/no health left in the health bar**. The player can continue playing and if [Sudden Death (SD)][Sudden Death wikilink]/[Perfect (PF)][Perfect wikilink] was not used, refilling the health bar completely will "revive" the player from the failed status to normal. Also, score will not be contributed towards the team score at this state. Text colour will be changed from white to red.
+**Tag/Green**         | _Tag Coop and Tag Team Vs only_. **This is a permanent colour scheme for the match to all and will not change based on health bar status**. There will be an arrow (`play-warningarrow`) pointing the current player playing the beatmap and given a small white glow.
+**Skipped/White**     | **A request notice from the player to skip the start-up rest if the beatmap/difficulty has a start-up rest**. Players that press the `Skip` button at the lower-right of the screen will be given a small yellow box at the lower right on their box with white Skipped text. **All players must do so for the Skip to take effect**.
+**Quit/Varies**       | **Player quit the match**. Two possible ways to enter this state: [1] hitting `Esc` twice, or [2] disconnected from Bancho. Text colour will be changed from white to red, with \[Quit\] appended at the end. Box colour signify the status before quitting the game.
+
+### Head to Head
+
+#### Interface
+
+![Head to Head Interface][HTH Interface image]
+
+_Head to Head interface_
+
+Head to Head is a team mode where players duke it all out in a free-for-all Match Setup for the top spot at the leaderboard.
+The default team mode to be selected and does not have any special characteristics in play; just go for the top spot.
+
+#### Result
+
+![Head to Head Grade screen][HTH Grade image]
+
+_Head-To Head grade screen_
+
+At the end of the match, first place player's avatar picture will be shown at the Winner section.
+If the player do not have one, it will simply be blank.
+
+Grades are now put inside the record like a watermark.
```

Comment:
???... `The grade icon is placed behind the winner image and behind the other players' results like a watermark.`